testdata/script/literals.txtar

```diff
@@ -330,7 +332,66 @@ func dotImportTest() {
     println(DotImportedFunc())
     println(DotImportedVar)
     println(DotImportedType("str as dot imported type"))
+    println(DotImportedStruct.Str)
 }
+
+func multipleTimeImportTest() {
+    regularAndUnusedRenamed()
+    regularAndUnusedDotImport()
+    unusedRegularAndDotImport()
+}
+
+func multipleCheck(vals ...string) {
+	if len(vals) < 2 {
+		return
+	}
+	for i := 0; i < len(vals)-1; i++ {
+		if vals[i] != vals[i+1] {
+			panic("vals must be equals")
+		}
+	}
+}
```

Comment:
not sure if this is necessary; if constant values were broken, our existing tests would already catch that :)